charts/netdata/templates/daemonset.yaml

```diff
@@ -95,6 +95,18 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ tpl .Values.image.tag . }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- with .Values.child.claiming }}
+          {{- if and .enabled .token .rooms }}
+          command:
+            - "/bin/sh"
+            - "-c"
+            - >-
+              exec /usr/sbin/run.sh
+                -W set2 cloud global enabled true
+                -W set2 cloud global "cloud base url" "{{ .url }}"
+                -W "claim -token={{ .token }} -rooms={{ .rooms }} -url={{ .url }}"
```

Comment:
Does it work without backslashes?

Reply:
Actually no. Thanks for spotting this! 🙇🏻 

It works if you remove the indentation. I have pushed it (tested it and it works)